src/ppx/knob.cpp

```diff
@@ -49,7 +49,6 @@ KnobCheckbox::KnobCheckbox(const std::string& flagName, bool defaultValue)
     : Knob(flagName, true), mValue(defaultValue), mDefaultValue(defaultValue)
 {
     SetFlagParameters("<true|false>");
-    RaiseUpdatedFlag();
```

Comment:
Removing `RaiseUpdatedFlag()` from different knob classes is going to affect the behaviour of all knobs. I don't think this PR adequately addresses that.

Reply:
Thanks for pointing it. I restored the calls to `RaiseUpdatedFlag()` and updated the PR.